bin/update_timestamp

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "specialist_publisher"
+
+def update
+  $stderr.puts %Q{
+USAGE: #{File.basename(__FILE__)} (all|[format ...])(content_id) (date)
+
+1. available formats:
+
+  #{SpecialistPublisher.document_types.sort.join("\n  ")}
+
+2. content_id:
+	To find content_id, login to specialist-publisher	in production, go to content item page and copy content_id from url.
```

Comment:
This isn't the `content_id` of the Document, as Specialist Documents don't live in the Content Store yet. This is better described as the `document_id`.